underscore.js

```diff
@@ -20,6 +20,7 @@
 
   // Save bytes in the minified (but not gzipped) version:
   var ArrayProto = Array.prototype, ObjProto = Object.prototype;
+  var SymbolProto = typeof Symbol !== 'undefined' ? Symbol.prototype : null;
```

Comment:
Hey @jdalton, curious what you meant by this

> the `undefined` check alone

and what a possible solution might be.